### DISCUSSION TITLE
131

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#63] [General]
# 潮流周刊第131期 - 大道芸人

封面来源于好多年前在大阪城公园拍的一个杂技艺人在表演，很是有趣，观众和表演者都是其乐融融的感觉。

https://weekly.tw93.fun/posts/131-%E5%A4%A7%E9%81%93%E8%8A%B8%E4%BA%BA/

**Comment by icm-ai** [comment]
让 GPT 知晓自己真正的版本，以及当前时间这个很有意思。

**Comment by eric2go** [comment]
看看自己头像是啥
